README.md

```diff
@@ -201,3 +200,21 @@ request.AddHeader("Content-Type", "application/json");
 request.AddParameter("undefined", "{\n\t\"to\" : \"+250781234567\", \n\t\"text\" : \"Hello from Pindo\",\n\t\"sender\" : \"Pindo\"\n}", ParameterType.RequestBody);
 IRestResponse response = client.Execute(request);
 ```
+
+```ruby
+# ruby
+
+require 'net/http'
+require 'json'
+
+uri = URI('http://api.pindo.io/v1/sms/')
+data = { to: '+250781234567', text: 'Hello from Pindo', sender: 'Pindo' };
+res = Net::HTTP.start(uri.host, uri.port) do |http|
+  req = Net::HTTP::Post.new(uri)
```

Comment:
Since we don't need SSL. I think it should better remove the `do` iteration.
```ruby
res = Net::HTTP.new(url.host, url.port);
req = Net::HTTP::Post.new(url)
```

Reply:
yeahh actually, I didn't think of that.